setup.cfg

```diff
@@ -67,6 +67,9 @@ net =
   zeep>=3.4.0
 timeseries =
   h5netcdf>=0.8.1
+  # While a not direct dependacy
+  # We need to raise this to ensure the goes netcdf files open.
+  h5py>=3.1.0
```

Comment:
So I wanted to pin to a >2 version but in 3.1 they fixed a bug with opening data with big endian and I felt that was worth pinning to as a min.